src/nyc_trees/apps/survey/templates/survey/reserve_blockface.html

```diff
@@ -38,6 +38,107 @@
 
 {% endblock main %}
 
+{% block extra_content %}
+<div class="modal fade" id="limit-reached-modal">
```

Comment:
I thought this would be easier for the design team to customize rather than rebuilding the JS bundle.